documentation/CAMARA-Security-Interoperability.md

```diff
@@ -0,0 +1,260 @@
+
+> Upon receipt of a scope parameter containing the offline_access value, the Authorization Server:
+
+>    MUST ensure that the prompt parameter contains consent unless other conditions for processing the request permitting offline access to the requested resources are in place; unless one or both of these conditions are fulfilled, then it MUST ignore the offline_access request,
+
+>    MUST ignore the offline_access request unless the Client is using a response_type value that would result in an Authorization Code being returned,
+
+>    MUST explicitly receive or have consent for offline access when the registered application_type is web,
+
+>    SHOULD explicitly receive or have consent for offline access when the registered application_type is native.
+
+The OIDC section on [OIDC Using Refresh Tokens](https://openid.net/specs/openid-connect-core-1_0.html#RefreshTokens) is not changed by this document.
+
+The OIDC security and privacy considerations regarding offline access and refresh tokens apply e.g. on
+[Token lifetime](https://openid.net/specs/openid-connect-core-1_0.html#TokenLifetime) and [Offline Access Privacy](https://openid.net/specs/openid-connect-core-1_0.html#OfflineAccessPrivacy).
+
+## Client-Initiated Backchannel Authentication Flow
+
+The [Client-Initiated Backchannel Authentication (CIBA)](https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html) flow is used to initiate an out-of-band authentication of the end-user.
+
+Communication with the Backchannel Authentication Endpoint MUST utilize TLS.
+
+CIBA allows the Client to get the authentication result in three ways: poll, ping, or push. This profile allows clients to use the *poll* mode. In the Poll mode, the authentication result is retrieved by the Client by polling the OP's token endpoint using the new grant type. 
+
+### Authentication Request
+
+This CAMARA document makes the login_hint parameter REQUIRED if CIBA is used in a CAMARA context.
+
+This CAMARA document clarifies the values used in login_hint in the following way:
+
+    * **_tel_**
+
+      For phone numbers. The `login_hint` must be a tel URI as defined in [RFC 3966](https://www.rfc-editor.org/info/rfc3966) for global phone numbers without visual separators in [E.164](https://www.itu.int/rec/T-REC-E.164-201011-I/en) format. For example, `tel:+34666666666`.
+
+    * **_ipport_**
+
+      For IPv4 and IPv6 addresses, that can optionally include a port. For example, `ipport:80.90.34.2:16790`, `ipport:80.90.34.2`, `ipport:[2001:db8::1]:8080` or `ipport:[2001:db8::1]`.
+
+
+The client MUST authenticate with the authorization server as described in [Client Authentication Section](#client-authentication).
+
+## Refresh Token Usage
+
+This section applies to OIDC Authorization Code Flow and CIBA. Other flows do not need refresh tokens and MUST not use refresh tokens.
```

Comment:
For clarity, we think that it is better to remove the _Offline Access_ section from _OIDC Authorization Code flow_ section and move the _Refresh Token Issuance_ section describing the use of `offline_access` scope, without the notes and duplicate text from the standard that could cause misunderstanding as to its intended use in the context of Camara:

```suggestion
## Refresh Tokens

This section applies to OIDC Authorization Code Flow and CIBA. Other flows do not need refresh tokens and MUST not use refresh tokens.

### Refresh Token Issuance

Neither OIDC not OAuth2 define a way for clients to indicate whether they need a refresh_token.  In OIDC and OAuth2 refresh token issuance is optional in Authorization Code Flow and CIBA, and at the discretion of the AZ.

Camara uses the scope `offline_access` in the authorization request to indicate to the AZ that the client requests a refresh token additionally to the access token for Camara API access.

### Refresh Token Usage
```